python/ql/src/Security/CWE-295/RequestWithoutValidation.ql

```diff
@@ -22,6 +22,6 @@ where
   // certificate validation, for example in `requests.get(..., verify=arg)`, `arg` would
   // be the `disablingNode`, and the `origin` would be the place were `arg` got its
   // value from.
-  if disablingNode = origin then ending = "." else ending = " by the value from $@."
-select request, "This request may run without certificate validation because it is $@" + ending,
-  disablingNode, "disabled here", origin, "here"
+  if disablingNode = origin then ending = "." else ending = " by $@."
+select request, "This request may run without certificate validation because $@" + ending,
+  disablingNode, "the request is disabled", origin, "this value"
```

Comment:
Why was this changed? What is the problem with saying that `This request may run without certificate validation because it is disabled`? -- If we want to follow the style guide, you can add `because it is disabled at [this location]` and make `this location` the link to the `disablingNode`

Reply:
I'm trying to avoid all kinds of "here" links. 

I'll try to go more towards the previous text, but adjusting where the links are. 
How about: `This request may run without certificate validation because [it is disabled].`
And the long version: `This request may run without certificate validation because [it is disabled] by [this value]`.